docs/content/preview/manage/ysql-major-upgrade-yugabyted.md

```diff
@@ -0,0 +1,225 @@
+headcontent: Upgrade YugabyteDB to a version that supports PG 15
+menu:
+  preview:
+    identifier: ysql-major-upgrade-1
+    parent: manage-upgrade-deployment
+    weight: 706
+type: docs
+---
+
+Upgrading YugabyteDB from a version based on PostgreSQL 11 (all versions prior to v2.25) to a version based on PostgreSQL 15 (v2.25.1 or later) requires additional steps. For instructions on upgrades within a major PostgreSQL version, refer to [Upgrade YugabyteDB](../upgrade-deployment/).
+
+The upgrade is fully online. While the upgrade is in progress, you have full and uninterrupted read and write access to your cluster.
+
+<ul class="nav nav-tabs-alt nav-tabs-yb">
+  <li>
+    <a href="../ysql-major-upgrade-yugabyted/" class="nav-link active">
+      <img src="/icons/database.svg" alt="Server Icon">
+      Yugabyted
+    </a>
+  </li>
+
+  <li>
+    <a href="../ysql-major-upgrade-local/" class="nav-link">
+      <i class="icon-shell"></i>
+      Manual
+    </a>
+  </li>
+
+</ul>
+
+## Before you begin
+
+{{< warning >}}
+v2.25 is a preview release that is only meant for evaluation purposes and should not be used in production.
+{{< /warning >}}
+
+- All DDL statements, except ones related to Temporary table and Refresh Materialized View, are blocked for the duration of the upgrade. Consider executing all DDLs before the upgrade, and pause any jobs that might run DDLs. DMLs are allowed.
+- Upgrade client drivers.
+
+    Upgrade all application client drivers to the new version. The client drivers are backwards compatible, and work with both the old and new versions of the database.
+- Your cluster must be running v2024.2.2.0 or later.
+
+    Deploy a new YugabyteDB cluster on version v2024.2.2.0 or later. If you have a pre-existing cluster, first upgrade it to the latest version in the v2024.2 series using the [upgrade instructions](../upgrade-deployment/).
+
+### Precheck
+
+New PostgreSQL major versions add many new features and performance improvements, but also remove some older unsupported features and data types. You can only upgrade after you remove all deprecated features and data types from your databases.
+
+Use the `upgrade check_version_compatibility` command to make sure your cluster is compatible with the new version.
+
+```sh
+./path_to_new_version/bin/yugabyted upgrade check_version_compatibility \
+    --base_dir=~/yugabyte-data/node1
+```
+
+```output
+output: ✅ Clusters are compatible for upgrade.
+```
+
+{{<tip title="Backup">}}
+Back up your cluster at this time. Refer to [Backup](../../reference/configuration/yugabyted/#backup).
+{{</tip>}}
+
+## Upgrade phase
+
+### Restart for catalog upgrade
+
+Restart the nodes one at a time as follows:
+
+```sh
+./path_to_new_version/bin/yugabyted stop --upgrade=true \
+    --base_dir=~/yugabyte-data/node1
+```
+
+```output
+Stopped yugabyted using config /net/dev-server-hsunder/share/yugabyte-data/node1/conf/yugabyted.conf.
+```
+
+```sh
+./path_to_new_version/bin/yugabyted start \
```

Comment:
In section - https://deploy-preview-26390--infallible-bardeen-164bc9.netlify.app/preview/manage/ysql-major-upgrade-yugabyted/#restart-for-catalog-upgrade

The yugabyted start command should not include the gflag `ysql_yb_major_version_upgrade_compatibility`. the command should be changed to the following - 

```
./path_to_new_version/bin/yugabyted start \
    --base_dir=~/yugabyte-data/node1 
```